backend/src/main/java/ai/verta/modeldb/versioning/RepositoryDAORdbImpl.java

```diff
@@ -510,7 +512,7 @@ private RepositoryEntity setRepository(
       }
     }
     session.getTransaction().commit();
-    if (create) {
+    if (create || nameChanged) {
```

Comment:
While updating repository name we should have to update name at UAC as well which is currently not happened so now fixed it with calling `setResource` if request found for update name